lib/model/tables/tunes2recordings.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:drift/drift.dart';
+
+class Tunes2Recordings extends Table {
```

Comment:
I think the traditional naming scheme would call the jump table TuneRecording. I think this is a bit better since in fact it represents bidirectional linkage, it can just as well go from recordings to tunes as tunes to recordings.